bin/serverless

```diff
@@ -2,26 +2,32 @@
 
 'use strict';
 
+const autocomplete = require('../lib/utils/autocomplete');
 const BbPromise = require('bluebird');
 const logError = require('../lib/classes/Error').logError;
 
-BbPromise.config({
-  longStackTraces: true,
-});
+if (process.argv[2] === 'completion') {
+  autocomplete();
+} else {
```

Comment:
@dougmoscrop hehe yeah that's what I did originally, but eslint complained that we're returning outside of a function 😒 ... So I thought I'd just follow the styler.